tests/unit/test_updater.py

```diff
@@ -83,7 +83,7 @@ def test_check_update_respect_config(mock_check, result, updater, mocker):
         ("0.0.2", "0.0.3", True),
         ("0.0.2", "0.0.1", False),
     ],
-    ids=["uptodate", "behind", "ahead"],
+    ids=["up-to-date", "behind", "ahead"],
```

Comment:
I'd prefer the previous one, as it is easier to type in the terminal, in case we need to run that particular test.

Reply:
Reverted.